esignet-core/src/main/java/io/mosip/esignet/core/dto/BindingOtpRequest.java

```diff
@@ -11,11 +11,13 @@
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import io.mosip.esignet.core.constants.ErrorConstants;
 import io.mosip.esignet.core.validator.OtpChannel;
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Why is this annotation added @Md-Humair-KK?

Reply:
Removed now